src/encord_active/server/routers/project.py

```diff
@@ -88,7 +129,13 @@ def tagged_items(project: ProjectFileStructureDep):
 
 @router.get("/{project}/items/{id:path}")
 def read_item(project: ProjectFileStructureDep, id: str):
-    lr_hash, du_hash, frame, *_ = id.split("_")
+    lr_hash, du_hash, frame, *object_hash = id.split("_")
+
+    row = get_model_prediction_by_id(project, id)
+
+    if row:
+        return to_item(row, project, lr_hash, du_hash, frame, object_hash[0] if len(object_hash) else None)
+
     with DBConnection(project) as conn:
         row = MergedMetrics(conn).get_row(id).dropna(axis=1).to_dict("records")[0]
```

Comment:
Perhaps we should add a fallback if there is nothing in this result. This is where we saw the app crashing previously.

Reply:
We should probably raise an informative error but the app should still fail.
The flow is "if there is a prediction with such id return it otherwise return a row from the merged metrics table".
If the id requested wasn't found anywhere we should probably return a 404.